addons/mail/models/discuss/discuss_channel.py

```diff
@@ -852,6 +882,8 @@ def _to_store(self, store: Store):
             info = channel._channel_basic_info()
             info["is_editable"] = channel.is_editable
             info["fetchChannelInfoState"] = "fetched"
+            info["parent_channel_id"] = Store.one(channel.parent_channel_id)
+            info["from_message_id"] = Store.one(channel.from_message_id)
```

Comment:
Message store is still sending full message data, is that necessary again?

Reply:
Yes. from_message_id will not be fetched from `/discuss/channel/messages` as it is part of another thread. If you want the thread info, you also want the from_message_id.